README.md

```diff
@@ -13,6 +25,8 @@ To get the encoding challenge for this one send a GET request to 192.168.22.100/
 
 `curl http://192.168.22.100/challenge1?whoAreYou=AHacker&areYouSure=yes`
 
+- **NOTE**: slashes are required in front of the `?` and `&` symbols (`\?` and `\&` respectively)
```

Comment:
I'm pretty sure I copied exactly what worked in my terminal for this answer. What terminal did you use for this?

Reply:
I'm using Alacritty on Manjaro Linux, if this is not standard in other terminals I'd be happy to remove or note the issue